### DISCUSSION TITLE
remote debugging ： “Bad owner or permissions” measure

### DISCUSSION THREAD

**Discussion by wilikeit** [pyscripter/pyscripter#1322] [Show and tell]
I think pyscripter is lightweight, powerful and easy to use.

【Environment】
win10 
pyscripter：Version 5.0.1.0 x64

【Problem】 
Bad owner or permissions 。

C:\Users\wilikeit\.ssh>ssh  root@192.168.1.226
Bad owner or permissions on C:\\Users\\wilikeit/.ssh/config    #My guess that owner or permissions  was altered by some other software.

 
### My solution：
# step1
create a new file : d:\tmp\config2    #I think the new created file belongs to the current user, the current user should have permission to access the config file

content:
Host 192.168.1.226
  HostName 192.168.1.226
  User root
  Port 22
  IdentityFile "C:\Users\wilikeit\.ssh\id_rsa"

try again，Successful！
ssh -F  d:\tmp\config2  root@192.168.1.226
Activate the web console with: systemctl enable --now cockpit.socket

Last login: Mon Sep 30 09:48:17 2024 from 192.168.1.102
Activate the web console with: systemctl enable --now cockpit.socket

Last login: Mon Sep 30 09:48:17 2024 from 192.168.1.102
[root@localhost ~]# exit  

 # setp 2
Run→ python  Engine→ssh
Tips：1、  add  "-F d:\tmp\config2"  2、You have the scp and ssh commands installed on your system.
scp option： -F d:\tmp\config2  -T -o PasswordAuthentication=no -o StrictHostKeyChecking=no
ssh option：  -F d:\tmp\config2  -o PasswordAuthentication=no -o StrictHostKeyChecking=no

**Comment by pyscripter** [comment]
I think the above is related to https://github.com/PowerShell/Win32-OpenSSH/wiki/Security-protection-of-various-files-in-Win32-OpenSSH (see at the bottom).  Please see the suggested solutions.

**Reply by wilikeit** [reply]
Thank you. Let me see this reference.
